test/client/fixtures/utils/dom-test.js

```diff
@@ -399,3 +399,19 @@ test('IsDomElement for plain object (T198784)', function () {
     strictEqual(eval(processScript('obj.target')), 'ok');
     /* eslint-enable no-unused-vars */
 });
+
+asyncTest('getParentWindowWithSrc (GH-114)', function () {
```

Comment:
Test name should describe the test case (good example: https://github.com/LavrovArtem/testcafe-hammerhead/blob/master/test/client/fixtures/sandbox/iframe-test.js#L175)